src/bot/lib/modifiers/register-commands.js

```diff
@@ -43,22 +43,14 @@ export function registerCommands(...commandsAndAliases) {
       })
 
       if (typeof result === 'string') {
-        return {
-          ...messageContext,
-          message: result,
-        }
+        return { message: result }
```

Comment:
I think it should set `fulfilled` at `true` at this point

Reply:
`fulfilled` is by default equal to `true`. See https://github.com/mindsers/sanpe-bot/pull/17/files#diff-f4fd88ee249fe934c063de2e756fd76d0a610ab3ad0068edb1dbbe870344b387R26